src/lib/components/video-player.svelte

```
@@ -0,0 +1,97 @@
+      <label for="quality-select">Quality:</label>
+      <select 
+        id="quality-select" 
+        bind:value={selectedIndex} 
+        onchange={(e) => changeQuality(+(e.target as HTMLSelectElement)?.value)}
+      >
+        <option value="-1">Auto</option>
+        {#each levels as level, i}
+          <option value={i}>
+            {level.height}p ({Math.round(level.bitrate / 1000)} kbps)
+          </option>
+        {/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)